Gemfile

```diff
@@ -51,6 +50,14 @@ gem 'sidekiq-statsd', '0.1.5'
 
 gem 'deprecated_columns', '0.1.0'
 
+if ENV['GDS_API_ADAPTERS_DEV']
```

Comment:
This is something I use and find helpful but I'm always very mindful when committing changes to both the `Gemfile` and `Gemfile.lock` through my own previous idiocy.